lib/main_common.pm

```diff
@@ -3190,4 +3196,26 @@ sub load_nfs_tests {
     loadtest "nfs/generate_report";
 }
 
+sub load_mm_nfs_test {
+    set_var('INSTALLONLY', 1);
```

Comment:
I would prefer this to be in test suite or job group level.

Reply:
That is actually a hack how to cut further test case loading. Not sure how to make it better/cleaner.

Reply:
But does it work as well if you add it to the static job variables?

Reply:
I guess, it should, frankly I haven't tried as it is kinda not really helpful to list it in among the variables. Mainly, this has been already there even before, so I haven't added it to this function, just to the JeOS subroutine in case the image should be published for further testing.